src/virtualenv/interpreters/create/creator.py

```diff
@@ -0,0 +1,182 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import json
+import logging
+import os
+import shutil
+import sys
+from abc import ABCMeta, abstractmethod
+from argparse import ArgumentTypeError
+
+import six
+from six import add_metaclass
+
+from virtualenv.info import IS_WIN
+from virtualenv.pyenv_cfg import PyEnvCfg
+from virtualenv.util.path import Path
+from virtualenv.util.subprocess import run_cmd
+from virtualenv.version import __version__
+
+HERE = Path(__file__).absolute().parent
+DEBUG_SCRIPT = HERE / "debug.py"
+
+
+@add_metaclass(ABCMeta)
+class Creator(object):
+    def __init__(self, options, interpreter):
+        self.interpreter = interpreter
+        self._debug = None
+        self.dest_dir = Path(options.dest_dir)
+        self.system_site_package = options.system_site
+        self.clear = options.clear
+        self.pyenv_cfg = PyEnvCfg.from_folder(self.dest_dir)
+
+    @classmethod
+    def add_parser_arguments(cls, parser, interpreter):
+        parser.add_argument(
+            "--clear",
+            dest="clear",
+            action="store_true",
+            help="clear out the non-root install and start from scratch",
+            default=False,
+        )
+
+        parser.add_argument(
+            "--system-site-packages",
+            default=False,
+            action="store_true",
+            dest="system_site",
+            help="Give the virtual environment access to the system site-packages dir.",
+        )
+
+        parser.add_argument(
+            "dest_dir", help="directory to create virtualenv at", type=cls.validate_dest_dir, default="env", nargs="?",
```

Comment:
One thing that occurred to me today -- after you pointed out this behavior to me -- I think we may consider that folks will want to put the default name that's chosen here in their gitignores under the presumption that that's someone creating a venv (I think for better or worse folks already occasionally do this for directories called `venv`).

Maybe as is it's still fine, but I have at least a suspicion that a slightly more specific name will satisfy that desire a bit more.

Is there a reason you didn't choose to name this `venv` by default, or was it arbitrarily / shorter to use `env`?

Reply:
mostly upon the assumption that's shorter; also python 2 virtual environments are not venvs, so I think helps say some-type of environment 🤔

Reply:
though maybe venv is better as it's a virtual environment 🤷‍♂ I'll change it to that